cookbooks/finite_strain/doc/finite_strain.tex

```diff
@@ -8,7 +8,7 @@ \subsubsection{Tracking finite strain}
 
 In many geophysical settings, material properties, and in particular the rheology, do not only depend
 on the current temperature, pressure and strain rate, but also on the history of the system.
-This can be incorporated in \aspect{} models by tracking history variables through compositional fields.
+This can be incorporated in ASPECT models by tracking history variables through compositional fields.
```

Comment:
Was this change on purpose? I dont think we need any changes to the .tex file?

Reply:
Definitely not